stargate-starter/src/main/java/io/stargate/starter/Starter.java

```diff
@@ -536,7 +546,9 @@ private void watchJarDirectory(String directoryPath) throws BundleException {
         poll = pollEvents(watchService);
       }
     } catch (IOException | ClosedWatchServiceException | InterruptedException e) {
-      throw new RuntimeException(e);
+      // Since most deployments will occur in the cloud its not worth breaking the start up to
+      // support hot-reload
+      System.err.printf("Jars will not be watched due to unexpected error: %s%n", e.getMessage());
```

Comment:
Since this is not via Logger system (probably since it's not configured yet), would it make sense to start with `ERROR:` prefix?

Reply:
https://github.com/stargate/stargate/issues/1303